### ISSUE TITLE
[Enhancement Request] Allow starting multiple servers with IDE

### ISSUE THREAD

**Issue by zhanghuabin** [akhikhl/gretty#189]
Hi, Andrey:

Sometimes I need to start several Gretty server instances within IDE at same time.

I can do this before 1.2.4(I changed servicePort and statusPort manually before starting each instance), but after upgrading to 1.2.4, you know, those two options are deprecated.

As workaround, I have to delete gretty_ports.properties before starting the new instance.

Could you please add a easier and configureable mechanism to allow starting multi-instances?

Thanks a lot
Huabin Zhang

**Comment by akhikhl** [comment]
do you mean you want multiple starts on the same project?

**Comment by zhanghuabin** [comment]
Yes, I sometimes start multiple servers of same project to simulate a cluster.

It will be very helpful if you can add an easier approach to allow starting multi-servers of same project at the same time.

Thanks a lot
:-)
